superset-frontend/src/dashboard/components/DashboardBuilder/state.ts

```diff
@@ -30,11 +30,6 @@ import {
 // eslint-disable-next-line import/prefer-default-export
 export const useNativeFilters = () => {
   const [isInitialized, setIsInitialized] = useState(false);
-  const showNativeFilters = useSelector<RootState, boolean>(
-    state =>
-      (getUrlParam(URL_PARAMS.showFilters) ?? true) &&
```

Comment:
Can you remove `showFilters` from `URL_PARAMS` or use JSDoc [@deprecated](https://jsdoc.app/tags-deprecated.html)?